emulator/templates/systemChunkTransactionTemplate.cdc

```diff
@@ -1,4 +1,4 @@
-import RandomBeaconHistory from 0xRANDOMBEACONHISTORYADDRESS
+import RandomBeaconHistory from "RandomBeaconHistory"
```

Comment:
Isn't the usual pattern `import "RandomBeaconHistory"`?

Reply:
yes, but we haven't gone through all the core contracts and updated everything to that format yet, so using this for now